docs/get-started/viam.md

```diff
@@ -15,143 +15,172 @@ imageAlt: "/general/understand.png"
 images: ["/general/understand.png"]
 ---
 
-Viam is a complete software platform that supports every step of your {{< glossary_tooltip term_id="smart-machine" text="smart machine">}} development lifecycle.
+Viam is a software platform that makes it easy to work with hardware and software.
+At the core of Viam is the open-source `viam-server` executable which:
```

Comment:
```suggestion

At the core of Viam is the open-source `viam-server` executable which:
```
See comment below but, at the very least I think it'd help to split this into a second paragraph so they digest the first simple sentence first